extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -135,6 +135,9 @@ private Containerizer createContainerizer(ContainerImageConfig containerImageCon
         ImageReference imageReference = getImageReference(containerImageConfig, applicationInfo);
 
         if (pushRequested || containerImageConfig.push) {
+            if (!containerImageConfig.registry.isPresent()) {
+                log.info("No container image registry was set, so 'docker.io' will be used");
```

Comment:
```suggestion
                log.info("No container image registry was set, defaulting to 'docker.io' ");
```

Reply:
I don't think this is an improvement to be honest